### PR TITLE
fixed extension

### PR DESCRIPTION
**Pull request by arpitvaghela**
Fix #5852

### CODE REVIEW

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py [hunk 1]
```diff
@@ -62,7 +53,7 @@ def set_classes(
 ) -> TypeAny:
 
     classes_set = set()
-    allowlist: TypeDict[str, str] = dict()
+    allowlist: TypeList[str,str] = []
```
Comment (Param-29) [inline packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py]: `allowlist` is a dictionary, always has been!

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py [hunk 2]
```diff
@@ -100,7 +91,7 @@ def set_classes(
                     if is_error:
                         debug_list.append(string)
                     else:
-                        allowlist[i + "." + t.__name__] = string
+                        allowlist.append((f"{i}.{t.__name__}",string))
```
Comment (Param-29) [inline packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py]: Same as above

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py [hunk 3]
```diff
@@ -188,12 +179,11 @@ def dict_allowlist(
     i: TypeAny,
 ) -> TypeAny:
 
-    # allowlist = {}
     methods_error_count = 0
     missing_return = 0
     # for i in classes_set:
     debug_list: TypeList[str] = list()
-    allowlist: TypeDict[str, str] = dict()
+    allowlist: TypeList[TypeTuple[str,str]] = []
```
Comment (Param-29) [inline packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py]: Same as above

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py [hunk 4]
```diff
@@ -237,15 +227,13 @@ def dict_allowlist(
                     )
 
                     list_nb.append(nbf.v4.new_code_cell(code))
-                allowlist[i + "." + t.__name__] = string
+                allowlist.append((f"${i}.${t.__name__}", string))
```
Comment (Param-29) [inline packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py]: Same as above

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py [hunk 5]
```diff
@@ -298,7 +286,7 @@ def main() -> None:
             missing_return_i,
             list_nb_i,
         ) = dict_allowlist(class_)
-        allowlist = {**allowlist, **allowlist_i}  # merging dicts
+        allowlist += allowlist_i # append to list
```
Comment (Param-29) [inline packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py]: Same as above

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py [hunk 6]
```diff
@@ -318,19 +306,16 @@ def main() -> None:
     package_support["lib"] = package_name
     # petlib doesnot have version
     # package_support["Version"] = package.__version__
-    package_support["class"] = list(classes_set)
+     # sort it all
+    classlist = list(classes_set)
+    classlist.sort()
```
Comment (Param-29) [inline packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py:311]: Q: Any specific reason who you are sorting?

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py [hunk 7]
```diff
@@ -342,6 +327,33 @@ def main() -> None:
...
+        "-l", dest="lib", required=True, help="name of the model to be added to ast"
+    )
+    parser.add_argument(
+        "-d", dest="debug", type=int, help="Set it to one to get debug files", default=0
+)
+    args = parser.parse_args()
+
+    DEBUG = args.debug
+    package_name = args.lib
+    generate_package_support(package_name, debug=DEBUG, write_to_file=True)
```
Comment (Param-29) [inline packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py:358]: Q: Any specific reason of changing the place of the code?

**Review by Param-29 — COMMENTED** [review]
Check it out

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py (hunk 6 above)
Comment (arpitvaghela) [reply packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py:311]: Madhava requested this for convenience while reading the json

File: packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py (hunk 7 above)
Comment (arpitvaghela) [reply packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py:358]: creating a argparser at module lvl causes problems with the extension my theory is it is caused as it overwrites / causes problems with the extension's arg parser

**Pull request closed without merging**